src/main/resources/forge-1.20.1/templates/livingentity/livingentity_renderer.java.ftl

```diff
@@ -105,6 +105,14 @@ public class ${name}Renderer extends <#if humanoid>Humanoid</#if>MobRenderer<${n
 				new HumanoidModel(context.bakeLayer(ModelLayers.PLAYER_OUTER_ARMOR)), context.getModelManager()));
 		</#if>
 
```

Comment:
Please follow formatting style we use accross codebase + merge both conditions, the if body is equivalent